drivers/volume/portworx/portworx.go

```diff
@@ -297,6 +310,46 @@ func (p *portworx) initPortworxClients() error {
 	return err
 }
 
+func (p *portworx) tokenGenerator() (string, error) {
```

Comment:
Add a comment here how it is expected that this code will be executed for those that are new to this model.

Reply:
done